### PR TITLE
Read a signed integer when using PUSHW in sanitizing a font (bug 1919513)

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: test/test_manifest.json [hunk 1]
```diff
@@ -10482,5 +10482,13 @@
     "rounds": 1,
     "link": true,
     "type": "eq"
+  },
+  {
+    "id": "bug1919513",
+    "file": "pdfs/bug1919513.pdf",
+    "md5": "83d1a7fab2c81e632910c334879e7334",
+    "rounds": 1,
+    "talos": false,
```
Comment (Snuffleupagus) [inline test/test_manifest.json:10491]: Any reason for this line?

**Review by Snuffleupagus — APPROVED** [review]
r=me, with one question; thank you.

File: test/test_manifest.json (hunk 1 above)
Comment (calixteman) [reply test/test_manifest.json:10491]: Yes, the people looking at the results are bit overwhelmed and they'd like to have less pdfs.
So we should try to reduce the number of them we've in Talos.
With this specific pdf, I don't think we'll have something significant to measure, so it's why I removed it from the list.
I'll try to take some time to check all of them and decide the pdfs we want to skip.

**Pull request merged**